bundles/org.openhab.binding.ekey/src/main/resources/OH-INF/i18n/ekey_de.properties

```diff
@@ -15,6 +15,8 @@ thing-type.config.ekey.protocol.label = Protokoll
 thing-type.config.ekey.protocol.description = Das vom ekey converter UDP verwendete Protokoll (RARE, MULTI oder HOME, abh�ngig vom verwendeten System).
 thing-type.config.ekey.delimiter.label = Abstandshalter
 thing-type.config.ekey.delimiter.description = Der im ekey converter UDP Protokoll verwendete Feldtrenner.
+thing-type.config.ekey.sourceIp.label = Quell IP
+thing-type.config.ekey.sourceIp.description = IPv4 Addresse eines eKey UDP Pakets. Kann abweichend von der ipAddress sein wenn NAT genutzt wird (z.B. in Kubernetes) 
```

Comment:
Wouldn't it be better to call this natIP instead of sourceIp ? In fact, it is not the source of the packets.